### PR TITLE
Docs - basic concepts

### PR DESCRIPTION
**Pull request by rahul1**

### CODE REVIEW

**Comment by rahul1** [comment]
WIP - draft of a "FHIR Basics Concepts" page for our docs. 
After finishing this, next step will be to create a "Medplum Basic Concepts" page

File: packages/docs/docs/fhir-basics.md [hunk 1]
```diff
@@ -0,0 +1,200 @@
...
+
+A core data object in FHIR is called a [**Resource**](https://www.hl7.org/fhir/resource.html). You can think of Resources as **objects** in object oriented languages. The FHIR standard defines a set of [**Resource Types**](/category/resources), similar to **classes**, that have been built for healthcare applications. A `Resource` is composed of multiple fields, called `Elements`, each of which can be a **primitive type** (e.g. strings, numbers, dates) or a **complex type** (e.g. JSON objects)
+
+Resources can represent a broad range of healthcare ideas, from very **concrete healthcare items** (e.g. [Patient](/api/fhir/resources/patient), [Medication](/api/fhir/resources/medication), [Device](/api/fhir/resources/device)) or more **abstract concepts** (e.g. [Procedure](/api/fhir/resources/procedure), [Care Plan](/api/fhir/resources/careplan), [Encounter](/api/fhir/resources/encounter)).
+
+### Example
+
+The example below shows an example **[Patient](/api/fhir/resources/patient) resource**, represented as JSON:
+
+```json
```
Comment (codyebberson) [inline packages/docs/docs/fhir-basics.md:19]: If you use type "javascript" or "ts" rather than "json", then you won't get errors for the comments.

**Review by codyebberson — APPROVED** [review]
LGTM 👍 

Minor tip about using "javascript" over "json" for comments, but feel free to merge.

File: packages/docs/docusaurus.config.js [hunk 2]
```diff
@@ -52,13 +52,28 @@ const config = {
...
             label: 'Docs',
+            position: 'left',
           },
           {
             to: '/blog',
             label: 'Blog',
             position: 'left',
           },
+          {
+            to: '/developers/intro',
```
Comment (codyebberson) [inline packages/docs/docusaurus.config.js:73]: This should be `/contributing/intro` now, yeah?

File: packages/docs/src/css/custom.css [hunk 3]
```diff
@@ -13,7 +13,7 @@
   --ifm-color-primary-light: #a36397;
   --ifm-color-primary-lighter: #a86a9c;
   --ifm-color-primary-lightest: #b581ab;
-  --ifm-font-size-base: 90%;
+  --ifm-font-size-base: 16px;
```
Comment (codyebberson) [inline packages/docs/src/css/custom.css:16]: I just checked, and the current size using `90%` is `14px`.  If I remember correctly, we reduced font size to accommodate the FHIR resource pages, which can have big tables with lots of data. 

I'm ok to tweak font size.  I just want to make sure we checked all of the impacts.

File: packages/docs/docs/fhir-basics.md (hunk 1 above)
Comment (codyebberson) [reply packages/docs/docs/fhir-basics.md:29]: Bump

**Pull request merged**